service/service_test.go

```diff
@@ -207,7 +140,7 @@ func TestConcurrentServiceStartAndShutdown(t *testing.T) {
 	t.Parallel()
```

Comment:
I wonder if `TestConcurrentServiceStartAndShutdown` is even a requirement for Poet?

Should a single service be used multiple times in the same running poet? Why? If an operator wants to run multiple poets on the same machine they can just run multiple instances of the program / docker container with different settings?

Reply:
It's required because `Start()` and `Shutdown()` might be called on different goroutines (Start - on a GRPC call, Shutdown - when the app closes).